app/views/lessons/show.html.erb

```diff
@@ -7,8 +7,29 @@
 
     <main class="grid grid-cols-12 gap-x-6" data-controller="lesson-toc" data-lesson-toc-item-classes-value="no-underline hover:text-gray-800 text-sm dark:hover:text-gray-300">
       <article class="col-span-full xl:col-span-7 xl:col-start-2">
-        <%= render ContentContainerComponent.new(classes: 'xl:mx-0', data_attributes: { lesson_toc_target: 'lessonContent' }) do %>
-          <%= @lesson.body.html_safe %>
+        <%= render ContentContainerComponent.new(classes: 'xl:mx-0', data_attributes: { lesson_toc_target: 'lessonContent' }) do |component| %>
+        <%= @lesson.body.html_safe %>
+
+        <% component.with_footer do %>
+          <div class="pt-10">
+          <%= link_to github_edit_url(@lesson), target: :_blank, rel: 'noreferrer noopener', class: 'underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-400 dark:hover:text-gray-300 px-5' do %>
+
+          <span class='fab fa-github mr-1'></span>
+
+          <span>
+            Improve on GitHub
+          </span>
+          <% end %>
+
+          <%= link_to github_report_url(@lesson), target: :_blank, rel: 'noreferrer noopener', class: 'underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-400 dark:hover:text-gray-300 px-5' do %>
```

Comment:
Sorry I missed this on last pass through, just needs that left alignment we talked about on call.

```suggestion
          <%= link_to github_report_url(@lesson), target: :_blank, rel: 'noreferrer noopener', class: 'underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-400 dark:hover:text-gray-300 pr-5' do %>
```

Last one, promise 😆